scripts/steps/00_setup_intel_fpga.source_bash

```diff
@@ -541,7 +541,7 @@ configure_fpga_quartus ()
 
         # Check the device name
 
-        if [ $device_name = 5CSE ] || [ $device_name = 5CSX ] || [ $device_name = 5CST ]
+        if [ $fpga_board = de10_nano ]
```

Comment:
Will it still work with the Cyclone V GX Starter Kit board?

Reply:
It will. Cyclone V GX Starter Kit board based on a trivial Cyclone V. Not a SoC.
Bitstream loads to the starter kit with the second pgm command. Without "@2".